app/src/main/java/org/simple/clinic/scheduleappointment/patientFacilityTransfer/PatientFacilityChangeActivity.kt

```diff
@@ -1,29 +1,70 @@
 package org.simple.clinic.scheduleappointment.patientFacilityTransfer
 
 import android.content.Context
+import android.content.Intent
 import android.os.Bundle
+import android.widget.RelativeLayout
 import androidx.appcompat.app.AppCompatActivity
+import androidx.interpolator.view.animation.FastOutSlowInInterpolator
+import com.jakewharton.rxbinding2.widget.RxTextView
+import com.mikepenz.itemanimators.SlideUpAlphaAnimator
 import io.github.inflationx.viewpump.ViewPumpContextWrapper
+import io.reactivex.Observable
+import io.reactivex.subjects.PublishSubject
+import kotlinx.android.synthetic.main.screen_patient_facility_change.*
 import org.simple.clinic.ClinicApp
 import org.simple.clinic.R
+import org.simple.clinic.bindUiToController
 import org.simple.clinic.di.InjectorProviderContextWrapper
 import org.simple.clinic.facility.change.FacilitiesUpdateType
+import org.simple.clinic.facility.change.FacilityChangeLocationPermissionChanged
+import org.simple.clinic.facility.change.FacilityChangeSearchQueryChanged
 import org.simple.clinic.facility.change.FacilityListItem
+import org.simple.clinic.location.LOCATION_PERMISSION
+import org.simple.clinic.registration.facility.FacilitiesAdapter
+import org.simple.clinic.router.ScreenResultBus
+import org.simple.clinic.router.screen.ActivityPermissionResult
+import org.simple.clinic.router.screen.ActivityResult
 import org.simple.clinic.util.LocaleOverrideContextWrapper
+import org.simple.clinic.util.RuntimePermissions
 import org.simple.clinic.util.wrap
+import org.simple.clinic.widgets.ScreenCreated
+import org.simple.clinic.widgets.ScreenDestroyed
+import org.simple.clinic.widgets.UiEvent
+import org.simple.clinic.widgets.displayedChildResId
 import java.util.Locale
 import javax.inject.Inject
 
 class PatientFacilityChangeActivity : AppCompatActivity() {
 
   @Inject
   lateinit var locale: Locale
-  
+
+  @Inject
+  lateinit var controller: PatientFacilityChangeController
+
+  private val recyclerViewAdapter = FacilitiesAdapter()
+
+  private val onDestroys = PublishSubject.create<ScreenDestroyed>()
+
+  private val screenResults = ScreenResultBus()
```

Comment:
The primary use of this class is to forward activity and permission results to screens when they are being managed with a `ScreenRouter`. Since this activity does not manage multiple screens, this is not necessary.